config/locales/client.en.yml

```diff
@@ -1510,11 +1509,6 @@ en:
             one: "1 person voted for this post"
             other: "{{count}} people voted for this post"
 
-      edits:
-        one: 1 edit
-        other: "{{count}} edits"
-        zero: no edits
```

Comment:
The `edits` [key exists for a long, long time](https://github.com/discourse/discourse/blame/master/config/locales/client.en.yml#L1513-L1516), but I couldn't find a place where it is actually used. That's why I removed it.